packages/graphql/lib/federation/graphql-federation.factory.ts

```diff
@@ -41,13 +41,16 @@ export class GraphQLFederationFactory {
 
   async mergeWithSchema<T extends GqlModuleOptions>(
     options: T = {} as T,
+    buildFederatedSchema?: (
+      options: import('@apollo/subgraph/dist/buildSubgraphSchema').GraphQLSchemaModule,
```

Comment:
Note: `import('@apollo/subgraph/dist/buildSubgraphSchema').GraphQLSchemaModule` this will make the `@apollo/subgraph` package required (even for those who don't use Federation in their apps).